examples/templates/aws-ecs-container/main.tf

```diff
@@ -11,16 +11,29 @@ terraform {
   }
 }
 
+provider "coder" {
+  feature_use_managed_variables = true
+}
+
 variable "ecs-cluster" {
   description = "Input the ECS cluster ARN to host the workspace"
   default     = ""
```

Comment:
I think that this variable should be required, so no default.

Reply:
it had a default set to emty so i just used that. Chnaged now.